packages/common-lib/src/components/theme.ts

```diff
@@ -71,11 +71,33 @@ const DEFAULT_THEME = {
         rounded: 'lg'
       },
       defaultProps: {
-        colorScheme: 'button'
+        colorScheme: 'button',
+        _text:{textTransform: 'uppercase', fontSize:'12px', fontWeight:'600'},
+
       }
     }
   },
   colors: {
+    widgetColor: {
+      400: '#7F9DAC',
```

Comment:
Can we please make theme name descriptive of what they do - 200 to 1000 is a little difficult to remember. They should map to a usecase - `highlightLight`, `highlightDark` etc.